mips_api/mips.py

```diff
@@ -0,0 +1,122 @@
+import os
+
+import boto3
+import requests
+
+
+class App:
+    _mips_url_login = 'https://login.abilaonline.com/api/v1/sso/mipadv/login'
+    _mips_url_chart = 'https://mipapi.abilaonline.com/api/v1/maintain/chartofaccounts'
+    _mips_url_logout = 'https://mipapi.abilaonline.com/api/security/logout'
+
+    ssm_client = None
+
+    required_secrets = [ 'user', 'pass' ]
+
+    def __init__(self):
+        if App.ssm_client is None:
+            App.ssm_client = boto3.client('ssm')
+
+        self._ssm_secrets = None
+        self._mips_org = None
+        self.mips_dict = {}
+        self.valid_routes = []
+
+        try:
+            self.valid_routes.append(os.environ['apiAllCostCenters'])
+        except KeyError:
+            raise Exception("The environment variable 'apiAllCostCenters' must be set.")
+
+        try:
+            self._mips_org = os.environ['MipsOrg']
+        except KeyError:
+            raise Exception("The environment variable 'MipsOrg' must be set.")
+
+        try:
+            self.ssm_path = os.environ['SsmPath']
+        except KeyError:
+            raise Exception("The environment variable 'SsmPath' must be set.")
```

Comment:
i think this can be simplified as ..

```
       self.expected_routes = ["apiAllCostCenters", "MipsOrg", "SsmPath"]
       for expected_route in self.expected_routes:
          try:
              self.valid_routes.append(os.environ['expected_route'])
          except KeyError:
              raise Exception(f'The environment variable {expected_route} must be set.')
```